tool/public/client.js

```diff
@@ -154,9 +154,12 @@ class FlagConverter {
 					case "srd5e":
 					case "plutonium":
 					case "core":
+					case "exportSource ":
```

Comment:
```suggestion
					case "exportSource":
```

Is this space intentional?

Reply:
ty, fixed